Tests/Model/ModelManagerTest.php

```diff
@@ -403,6 +404,77 @@ public function testAssociationIdentifierType()
         $this->assertSame(42, $result[0]);
     }
 
+    /**
+     * [sortBy, sortOrder, isAddOrderBy].
+     *
+     * @return array
+     */
+    public function getSortableInDataSourceIteratorDataProvider()
+    {
+        return array(
+            array(null, null, false),
+            array('', 'ASC', false),
+            array('field', 'ASC', true),
+        );
+    }
+
+    /**
+     * @dataProvider getSortableInDataSourceIteratorDataProvider
+     *
+     * @param string|null $sortBy
+     * @param string|null $sortOrder
+     * @param bool        $isAddOrderBy
+     */
+    public function testSortableInDataSourceIterator($sortBy, $sortOrder, $isAddOrderBy)
+    {
+        $datagrid = $this->getMockForAbstractClass('Sonata\AdminBundle\Datagrid\DatagridInterface');
```

Comment:
I saw this pull request. 
I think I can create mock from builder, and you don't need fixed unit test after merge pull request.  Is solution suitable?